docs/product-getting-started-guide/src/main/asciidoc/task_business-process.adoc

```diff
@@ -10,11 +10,6 @@ The following chapter instructs you how to create a business process in {PRODUCT
 
```

Comment:
Besides the typo at line 50, looks good to me. I also noted there another undesirable structuring issue that comes up occasionally in the steps, but not worth your time right now. Just brought it up.